compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -469,9 +469,11 @@ impl<'a, 'b> BuildReducedGraphVisitor<'a, 'b> {
                         }
 
                         // Replace `use foo::{ self };` with `use foo;`
+                        let source_span = source.ident.span;
```

Comment:
nit: I think `self_span` would be a better name, I found this a little confusing